lib/api_transaction.go

```diff
@@ -137,3 +138,73 @@ func GetTransactionByHashHandler(storage *sebakstorage.LevelDBBackend) http.Hand
 		}
 	}
 }
+
+const GetMissingTransactionByHashHandlerPattern = "/missingtxs"
+
+func GetMissingTransactionsByHashHandler(storage *sebakstorage.LevelDBBackend /**nr *NodeRunner**/) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		key := vars["hash"]
+		// var requestBody []byte
+		var err error
+
+		if r.Method != "POST" {
+			http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
```

Comment:
That's not the correct status though. Under no condition can a `POST` request to this endpoint be correct, so it should be a 400. This indicates "You are missing authentication".

Reply:
OK.